lib/templates/docs-list.hbs

```diff
@@ -2,11 +2,15 @@
 
     {{#*inline "content"}}
         <div class="docs-list">
+
             <div class="wrapper">
+
                 <ul>
+                    <h1>API Docs</h1>
                     {{#each list}}
                         <li>
```

Comment:
Just turn this into an `<a>` and the title into a `<span>` (or something like that).  The `<li>` doesn't really fit without an `<ol>` or `<ul>` surrounding it anyways.